tests/integration/v2/test_integration_v2.py

```diff
@@ -87,7 +87,7 @@ def test_given_charms_deployed_when_relate_then_status_is_active(
 def test_given_charms_deployed_when_relate_then_requirer_received_certs(
     juju: jubilant.Juju,
 ):
-    if juju.cli("version").startswith("2.9"):
+    if juju.cli("version", include_model=False).startswith("2.9"):
```

Comment:
You could potentially wrap this in a session-scoped fixture called `juju_version` or even `is_juju_29`.

Reply:
Yes, that's a good suggestion. I thought about doing something like that, but had to make everything pass on 2.9 first. I think I finally got it, and the code is not too awful to support 2.9.